x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/configuration_step/configuration_step_form.tsx

```diff
@@ -69,17 +70,20 @@ export const ConfigurationStepForm: FC<CreateAnalyticsStepProps> = ({
   const [dependentVariableOptions, setDependentVariableOptions] = useState<
     EuiComboBoxOptionOption[]
   >([]);
-  const [excludesTableItems, setExcludesTableItems] = useState<FieldSelectionItem[]>([]);
+  const [includesTableItems, setIncludesTableItems] = useState<FieldSelectionItem[]>([]);
   const [maxDistinctValuesError, setMaxDistinctValuesError] = useState<string | undefined>(
     undefined
   );
+  const [unsupportedFieldsError, setUnsupportedFieldsError] = useState<string | undefined>(
+    undefined
+  );
```

Comment:
Nit: AFAIK for the intialization with `undefined` you can just to `()` instead of `(undefined)`.